src/App.js

```diff
@@ -68,7 +68,7 @@ function App() {
               <div className="copied">{copied ? 'copied' : ''}</div>
             </section>
             <section>
-              <WiredButton elevation={2} onClick={() => reGenerate()}>
+              <WiredButton elevation={2} onClick={reGenerate}>
```

Comment:
shouldn't be this called on render?

Reply:
Not quite, I think these 2 are the same?

```js
() => reGenerate()
reGenerate
```

Since `reGenerate` itself is a function, if unpacked, it will become like this:

```
() => (() => { /*  reGenerate function content */ })()
() => { /* reGenerate function content */ }
```

Sample sandbox: https://codesandbox.io/s/affectionate-newton-pvv9s?file=/src/App.tsx

Reply:
ah right, sure